src/test/groovy/life/qbic/datamodel/datasets/datastructure/OxfordNanoporeExperimentSpec.groovy

```diff
@@ -37,16 +43,20 @@ class OxfordNanoporeExperimentSpec extends Specification {
     Map minimalWorkingPooledDataStructure
 
     def setupSpec() {
-        InputStream stream = Thread.currentThread().getContextClassLoader().getResourceAsStream("valid-example.json")
+        def folder = "nanopore/"
+        InputStream stream = Thread.currentThread().getContextClassLoader().getResourceAsStream(folder+"valid-example.json")
         minimalWorkingSimpleDataStructure = (Map) new JsonSlurper().parse(stream)
         // new example with slightly different structure
-        stream = Thread.currentThread().getContextClassLoader().getResourceAsStream("valid-example-newer.json")
+        stream = Thread.currentThread().getContextClassLoader().getResourceAsStream(folder+"valid-example-v2.json")
+        extendedWorkingSimpleDataStructureWithReportsFolder = (Map) new JsonSlurper().parse(stream)
```

Comment:
Nit: Same as the previous comment, instead of v2 i'd argue that a naming schema highlighting the distinction between both schemas/data structures would make it more readable.